spec/factories/payment.rb

```diff
@@ -0,0 +1,8 @@
+FactoryBot.define do
+  factory :payment do
+    mollie_id { Faker::Name.initials(number: 4) }
+    amount { rand(20..50) }
```

Comment:
Can this range be larger to also test very tiny and very large amounts?

Reply:
I've increased it too make larger amounts. Smaller is not accepted since 20 is the minimum